packages/ui/components/test-setup.ts

```diff
@@ -1,6 +1,16 @@
 import matchers from "@testing-library/jest-dom/matchers";
 import { cleanup } from "@testing-library/react";
-import { afterEach, expect } from "vitest";
+import { afterEach, expect, vi } from "vitest";
+
+vi.mock("@calcom/lib/OgImages", async () => {
+  return {};
+});
```

Comment:
We added it to solve the following error:
![image](https://github.com/calcom/cal.com/assets/121884634/51f7adda-69bc-437a-b68c-5ac5ff9b29bd)